druid/src/win_handler.rs

```diff
@@ -339,7 +339,8 @@ impl<T: Data> Inner<T> {
             }
             Target::Global => {
                 for w in self.windows.iter_mut() {
-                    let event = Event::Command(cmd.clone());
+                    let event =
+                        Event::Internal(InternalEvent::TargetedCommand(target, cmd.clone()));
                     if w.event(&mut self.command_queue, event, &mut self.data, &self.env) {
                         break;
                     }
```

Comment:
Not related to your PR but made me wonder:
Does it make sense to allow global commands to be 'handled'?
It seems wrong that global commands, which IMO are supposed to reach everywhere, can be cough by some widget, in a not necessarily deterministic way (as it depends on the window order).

Reply:
This touches upon the whole `handled` concept. A similar problem applies to all the other events as well.

Setting `handled` is never that deterministic, global or not. I could also see it working on a per-window basis for global commands. Either way it would probably be good to document that in the docs of `Target`.